ee/clickhouse/sql/events.py

```diff
@@ -179,7 +179,7 @@
     events_with_array_props_view ewap
 where ewap.team_id = %(team_id)s
 AND ewap.uuid IN (select uuid from events WHERE team_id = %(team_id)s {conditions})
-ORDER BY ewap.timestamp DESC {limit}
+ORDER BY toDate(ewap.timestamp) DESC, ewap.timestamp DESC {limit}
```

Comment:
tests work now! Needed to specify the direction of ordering on the todate(timestamp) also. Somehow there was a difference between the dates being inserted on github CI and locally which caused this.

Reply:
Hero! Thanks very much